src/AzureClient/IAzureClient.cs

```diff
@@ -65,19 +65,21 @@ public interface IAzureClient
     {
         /// <summary>
         /// Connects to the specified Azure Quantum workspace, first logging into Azure if necessary.
+        /// Returns the list of execution targets available in the Azure Quantum workspace.
```

Comment:
Very minor, but this may be good to include in a `<returns>` element.

Reply:
Thanks, I'll update all the comments in this file.